src/protocols/abstract/Realtime.ts

```diff
@@ -83,11 +83,12 @@ export abstract class BaseProtocolRealtime extends KuzzleAbstractProtocol {
       this.retrying = true;
 
       if (
+        window !== null &&
```

Comment:
I don't really understand where is the actual fix. It seems that this line is the only addition to the codebase :thinking:

Reply:
Fixing the issue did not require an addition but a deletion.
The `from` and `size` parameters were sent both in the body and in the query params via the `options` property of query,  the catch is that the values in the body were different that the ones in the query params, the values in the query params are directly copied from the original query (because we wanted to keep the same options as the original query) but this had the effect to override the new values that were put in the body, causing the request to always be the same no matter what has been put in the body.